docs/quick-start/vote-contract/develop-smart-contract.md

```diff
@@ -248,6 +248,10 @@ namespace AElf.Contracts.BuildersDAO
 }
 ```
 
+:::danger
+Aelf sidechain does not recognise duplicate smart contracts hence we will be using the author variable as the unique identifier for our voting smart contract.
```

Comment:
To be more explicit:
Aelf sidechain does not allow duplicate identical smart contracts. Hence, we will be using the author variable as the unique identifier for our voting smart contract in order to deploy the smart contract successfully.